generate-docs/tools/version-remover.ts

```diff
@@ -45,10 +52,10 @@ fsx.writeFileSync(process.argv[4], wholeDTS);
 
 
 function getDeclarationEnd(wholeDts: string, startIndex: number): number {
```

Comment:
Minor suggestion: For consistency in using the camelCase naming convention, consider also updating the `wholeDts` variable name declared in line 10.